funcs/historical.js

```diff
@@ -147,8 +145,8 @@ const getHistoricalDataV2 = (data, lastdays = 30) => {
  * @returns {Object}				The filtered historical data.
  */
 const getHistoricalCountryDataV2 = (data, query, province = null, lastdays = 30) => {
-	if (lastdays === 'all') lastdays = 0;
-	if (isNaN(lastdays)) lastdays = 30;
+	if (lastdays && lastdays === 'all') lastdays = Number.POSITIVE_INFINITY;
+	if (!lastdays || isNaN(lastdays)) lastdays = 30;
```

Comment:
instead of an if, just make this else statement

Reply:
Ethan no